### PR TITLE
Embed swagger-ui to mkdocs

### PR DESCRIPTION
**Pull request by sherenewso2**
## Purpose
- Add swagger library
- Add a template for swagger docs
- Add swagger yaml files of 5.9.0 rest apis
- Edit theme.css to hide swagger topbar
- Add swagger-ui colour themes

### CODE REVIEW

File: en/docs/assets/css/theme.css [hunk 1]
```diff
@@ -857,3 +857,6 @@ html .md-footer-meta.md-typeset a,
     background-color: #e8e8e8;
     color: #4d4d4d;
 }
+.swagger-ui .topbar {
```
Comment (jeradrutnam) [inline en/docs/assets/css/theme.css]: Keep a new line after `}`

File: en/theme/material/templates/swagger.html [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+-->
+
+{% extends "base.html" %}
+
+{% block styles %}
+  {{ super() }}
+  <link rel="stylesheet" type="text/css" href="../../assets/lib/swagger/swagger-ui.css" >
```
Comment (jeradrutnam) [inline en/theme/material/templates/swagger.html]: Will the same path works if you move it to live?

File: en/docs/assets/css/theme.css (hunk 1 above)
Comment (sherenewso2) [reply en/docs/assets/css/theme.css]: Fixed with 884fb2716c42398efa8ebaecb1f154e216d5f0fa

File: en/theme/material/templates/swagger.html (hunk 2 above)
Comment (sherenewso2) [reply en/theme/material/templates/swagger.html]: This is the same way we give image paths and links to other pages. It should probably work in the live site.

Comment (sherenewso2) [reply en/theme/material/templates/swagger.html]: Fixed with e6b991de6d60de17e6b272d282e7b90debdec42d

**Pull request merged**